wazo_webhookd/services/mobile/plugin.py

```diff
@@ -79,7 +80,10 @@ def on_external_auth_added(self, body, event):
                     'owner_user_uuid': user_uuid,
                     'owner_tenant_uuid': tenant_uuid,
                     'config': {},
-                    'metadata_': {'mobile': 'true'},
+                    'metadata_': {
+                        'mobile': 'true',
+                        'subscription_token': self.subscription_token,
```

Comment:
Nice idea! Unfortunately fblackburn confirmed to me that the subscription token is not static: subscription tokens expire and may be renewed. In this case, the push notification will still be sent with the old expired subscription token, since the webhookd subscription will not be updated. I think we have to cache the subscription token and check that it is not expired each time we use it, then update it if it is expired.

Reply:
Also, it's missing a config option to disable getting the subscription token feature.

Reply:
But you told me they don't expire ! :-) Just kidding, I'll update my PR